reel-picker/src/main/java/com/innowisegroup/reelpicker/picker/ui/TimePickerFragment.kt

```diff
@@ -38,17 +38,15 @@ internal class TimePickerFragment : Fragment() {
         savedInstanceState: Bundle?
     ): View? {
         val view = inflater.inflate(R.layout.item_time_picker, container, false)
-        localDateTime = requireArguments().getSerializable(LOCAL_TIME) as? LocalDateTime
-        minDateTime = requireArguments().getSerializable(MIN_TIME) as? LocalDateTime
-        maxDateTime = requireArguments().getSerializable(MAX_TIME) as? LocalDateTime
+        applyArguments()
```

Comment:
The same about methods strucure

applyArguments()
applyAttributes()
setFragmentResultListeners()
initializeView(view)